tools/phenotype_annotation/phenotype_annotation

```diff
@@ -0,0 +1,351 @@
+                                              -start => $vfStart,
+                                              -end => $vfEnd,
+                                              -strand => 1,
+                                              -seq_region_name => $vfChrom,
+                                              -seq_region_length => $vfEnd - $vfStart + 1,
+                                              -adaptor => $sa);
+      # get phenotypes for genes that are overlapped by query variation
+      my @extracted_phenos = @{$pfa->fetch_all_by_Slice_accession_type($tmpSlice, $mapping_type)};
+      warn "INFO: found ", scalar @extracted_phenos, " phenotypes assigned to variation $name\n" if $verbose;
+      my %phenotypes = map { $_->phenotype_id. $_->seq_region_name. $_->seq_region_start.$_->seq_region_end => $_ } @extracted_phenos; #make phenotype features unique
+      @extracted_phenos = values %phenotypes;
+      print_phenos($name, \@extracted_phenos, $outType);
+    }
+    $doneList{$record} = 1;
+  }
+  $parser->close();
+}
+
+sub print_phenos {
+  my ($uploaded_name, $phenof, $format) = @_;
+  foreach my $ph (@$phenof){
+    my $coords =  join("-",$ph->seq_region_name, $ph->seq_region_start, $ph->seq_region_end);
+    my $acc_tmp = join(";", @{$ph->get_all_ontology_accessions});
+    my $desc =  $ph->phenotype_description();
+    if ($format eq 'bed') {
+      print $outFH join("\t", $ph->seq_region_name, $ph->seq_region_start, $ph->seq_region_end,
+                   join(":", $ph->object_id, "'".$desc."'(". $acc_tmp .")")
+                   ) , "\n";
+    } elsif ($format eq 'vcf') {
+      my $tmp ="."; # for REF, ALT, QUAL, FILTER
+      print $outFH join("\t", $ph->seq_region_name, $ph->seq_region_start, $ph->seq_region_end,
+                        $tmp, $tmp, $tmp, $tmp,
+                        "OBJ=".$ph->object_id.";" . "PHEN=".join("|",$desc, $acc_tmp)
+                        ) , "\n";
+    } else {
+      print $outFH join("\t", $uploaded_name, $ph->seq_region_name, $ph->seq_region_start, $ph->seq_region_end, $ph->strand,
+                   $ph->object_id, $desc, $acc_tmp), "\n";
+    }
+  }
+}
+
+sub usage {
+
+  print qq{
+  Help: dev\@ensembl.org , helpdesk\@ensembl.org
+  Usage:  ./phenotype_annotation [OPTIONS]
+  Annotate input data: variants (list of rsIDs or VCF or VEP output file) OR genes (list of gene identifiers or gene symbols) OR phenotypes (list of accessions or term names) with assigned phenotype features.
+  Options:
+      -input_file|i      Input file (variants in VCF, rsIDs or genes or phenotypes in text file list form)
+      -input_type        Type of input data: [variant, gene, phenotype, vep] (default: gene)
+
+      -species           Species of the input data (default: human)
+      -assembly          Assembly to use if species is human (default: grch38)
+      -registry          Ensembl registry file containing database connection details
```

Comment:
One last thing - it's not clear that you don't need to supply a registry file, which may confuse people. Putting a "(default: uses current Ensembl release database)" would clarify.